src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+    private final int hypotenuse;
```

Comment:
The hypotenuse is not needed as a field in this class. It's not used for the area calculation, nor is it relevant for a `RightTriangle` object. Consider removing it to simplify the class.